transaction/core/src/blockchain/block_signature.rs

```diff
@@ -21,6 +21,11 @@ pub struct BlockSignature {
     /// The public key of the keypair used to generate the signature.
     #[prost(message, required, tag = "2")]
     signer: Ed25519Public,
+
+    /// An approximate time in which the block was signed.
+    /// Represented as seconds of UTC time since Unix epoch 1970-01-01T00:00:00Z.
```

Comment:
maybe add a note that, this should not be included in hashes or something?

Reply:
Thats true for the entire signature.